File: URI_1042_simple_sort.js
const lines = ["7 21 -14"];

const inputArr = lines[0].split(" ");
const a = Number(inputArr[0]);
const b = Number(inputArr[1]);
const c = Number(inputArr[2]);

const arrToSort = [a, b, c];
arrToSort.sort(function(a, b){return a - b});

console.log(arrToSort[0]);
console.log(arrToSort[1]);
console.log(arrToSort[2]);
console.log();
console.log(a);
console.log(b);
console.log(c);

File: URI_1042_simple_sort.java
import java.util.Arrays;
import java.util.Scanner;

public class URI_1042_simple_sort {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int a = scanner.nextInt();
		int b = scanner.nextInt();
		int c = scanner.nextInt();
		
		int[] inputArr = {a, b, c};
		Arrays.sort(inputArr);
		
		System.out.println(inputArr[0]);
		System.out.println(inputArr[1]);
		System.out.println(inputArr[2]);
		System.out.println();
		System.out.println(a);
		System.out.println(b);
		System.out.println(c);

		scanner.close();
	}
}